adafruit_scd4x.py

```diff
@@ -204,14 +209,14 @@ def _read_data(self):
         self._relative_humidity = 100 * (humi / 2**16)
 
     @property
-    def data_ready(self):
+    def data_ready(self) -> bool:
         """Check the sensor to see if new data is available"""
         self._send_command(_SCD4X_DATAREADY, cmd_delay=0.001)
         self._read_reply(self._buffer, 3)
         return not ((self._buffer[0] & 0x07 == 0) and (self._buffer[1] == 0))
 
     @property
-    def serial_number(self):
+    def serial_number(self) -> bytearray:
```

Comment:
sorry, been out of town all week.

Reply:
I see from the docstring that it returns a six-tuple:  what ?   pylint is NOT liking Tuple(int, int, int, int, int, int) or tuple(int, int, int, int, int, int).  It doesn't like Tuple at all, and says that def serial_number(self) -> tuple(int, int, int, int, int, int):
TypeError: tuple expected at most 1 argument, got 6

I feel like I left my brain in Michigan.....

Reply:
Note the square brackets `Tuple[int, int, int, int, int, int]` instead of `()`, that's the issue.  And make sure to use `Tuple` and not `tuple` since this needs to be able to run on Python 3.7.

Reply:
yep, I had the flash of memory just as you sent that.......like I said, I left my brain in indiana